app/selenium_ui/bamboo/modules.py

```diff
@@ -31,8 +34,102 @@ def sub_measure():
     measure()
 
 
+def view_all_projects(webdriver, datasets):
+    @print_timing("selenium_view_all_projects")
+    def measure():
+        projects_page = ProjectList(webdriver)
+        projects_page.click_projects_button()
+        projects_page.wait_for_page_loaded()
+    measure()
+
+
+def view_all_builds(webdriver, datasets):
+    @print_timing("selenium_view_all_builds")
+    def measure():
+        builds_page = BuildList(webdriver)
+        builds_page.click_all_builds_button()
+        builds_page.wait_for_page_loaded()
+
+    measure()
+
+
+def config_plan(webdriver, datasets):
+    @print_timing("selenium_config_plan")
+    def measure():
+        config_page = PlanConfiguration(webdriver)
+        config_page.click_config_plan_button()
+        config_page.wait_for_page_loaded()
+
+    measure()
+
+
+def builds_queue_page(webdriver, datasets):
```

Comment:
Lets have the naming like in @print_timing decorator but without 'selenium_'. So there should be "view_build_activity". And, please, double-check the same in other places